### ISSUE TITLE
Do you even know how MCP works

### ISSUE THREAD

**Issue by catfoolyou** [EaglerDevs/Eaglercraft-1.11.2#1]
Also why are you doing stuff in Python, just learn Java at this point

**Comment by SchoolTARMaster** [comment]
true. the languages: python. wtf? neither minecraft java edition nor eaglercraft use python. you need to know java. and if you think you are smart what does MCP stand for (no searching) ? bet you'll say mInE CrAfT PoCkEt.

**Comment by catfoolyou** [comment]
These guys are skids and liars

**Comment by MohHossary** [comment]
guys he just needs to learn
BTW @RaveTek ill teach u just dm me on dc MohAlhossary

**Comment by catfoolyou** [comment]
He's a SKID, he isn't gonna learn anything. He'll just steal more stuff and claim that he made it

**Comment by MohHossary** [comment]
nah i feel like hes gonna be ok
BTW @RaveTek Do u want me to teach u?
And @catfoolyou stop calling ppl skids

**Comment by MohHossary** [comment]
BTW @SchoolTARMaster he literally says the meaning of MCP in the desc of the repo what are you talking abt

**Comment by MohHossary** [comment]
And I also realized in teavm.org it says; "TeaVM is an ahead-of-time **compiler** for Java bytecode that emits JavaScript and WebAssembly"
it is a compiler that instead of producing java bytecode, takes in java bytecode and emits javascript and wasm.
So @RaveTek can technically write it in javascript if he wants to.
So stop bullying everyone who tries making an eaglercraft client please

**Comment by catfoolyou** [comment]
These people have been consistently making fake eagler ports and stealing other people's work @MohHossary 
So technically they are skids

**Comment by catfoolyou** [comment]
And what exactly will you teach him? Java and the way TeaVM works?

**Comment by MohHossary** [comment]
yea actually how the jvm works and how to use mcp (hence the name of the issue)

**Comment by SchoolTARMaster** [comment]
@MohHossary  did you know that I DON'T GIVE A THING and RaveTek is just skidding like go to https://github.com/RaveTek and look at a 404 bro's account either got deleted or he deleted it because bro is sad, mad, broken on his bed, hoping he gets rid of all the voices in his head.
And @MohHossary  you're not الشيخ الحصري so stop acting.

**Comment by SchoolTARMaster** [comment]
And how are you going to teach him if his account is deleted ?

**Comment by catfoolyou** [comment]
> @MohHossary did you know that I DON'T GIVE A THING and RaveTek is just skidding like go to https://github.com/RaveTek and look at a 404 bro's account either got deleted or he deleted it because bro is sad, mad, broken on his bed, hoping he gets rid of all the voices in his head. And @MohHossary you're not الشيخ الحصري so stop acting.

Ravetek got banned

**Comment by SchoolTARMaster** [comment]
@catfoolyou but why who banned him ?

**Comment by SchoolTARMaster** [comment]
Also @catfoolyou can you go <a href="https://www.reddit.com/r/eaglercraft/comments/1hh9g8o/how_did_someone_who_catfoolyou_hated_and_called_a/">here</a> can you respond maybe ?

**Comment by SchoolTARMaster** [comment]
Also, let me prove how skiddy RaveTek is.
So, basically he has an account on Planet Minecraft, named TC-The Cat
And he posted something under the Mods category, called Ravencraft 1.21. Ravencraft 1.21 is just something based on EaglercraftX 1.8.8 RC-01 which he created that edits four things: Some textures, the Mojang logo in the loading screen, the name in the bottom left of the title screen, and the background of the title screen. That's okay, but the thing is, HE ONLY PROVIDES THE ONLINE DOWNLOAD AND DOESN'T EVEN TRY TO MAKE THE OFFLINE DOWNLOAD??!! IT WAS LIKE 4 MONTHS SINCE HE POSTED THAT THING ON PLANET MINECRAFT. I made the offline download, not in the normal script order and without a timer, check it out <a href="https://gitlab.com/TARMaster/ravencraft1-21">here</a>.
